go.mod

```diff
@@ -3,16 +3,27 @@ module github.com/scylladb/gocqlx/v2
 go 1.17
 
 require (
-	github.com/gocql/gocql v0.0.0-20211015133455-b225f9b53fa1
+
+	github.com/gocql/gocql v1.0.0
 	github.com/google/go-cmp v0.5.4
+	github.com/maraino/go-mock v0.0.0-20180321183845-4c74c434cd3a
```

Comment:
If possible also this extra dependency: `github.com/maraino/go-mock`

Reply:
This dependency is added because I am using a the go-mock library to mock the session on unit tests

Reply:
If I get rid of it then I am not sure about the way to mock the session on unit tests.

Reply:
I removed the tests as they were not really testing anything anyway.

All parts that are used are tested in other unit tests already. We are not really adding something worth testing.